### PR TITLE
Link to Master Preview/Main Preview was broken in TOC

### PR DESCRIPTION
**Pull request by ValGrimm-U3D**
The link to Master Preview/Main Preview has been broken since 9.0. I updated the name of the relevant page in the TOC.

# **Please read;**
## **PR Workflow for the Graphics repository:**
* **All PRs must be opened as draft initially**
* Reviewers can be added while the PR is still in draft
* The PR can be marked as “Ready for Review” once the reviewers have confirmed that **no more changes are needed**
* Tests will start automatically after the PR is marked as “Ready for Review”
* **Do not use [skip ci]** - this can break some of our tooling
* Read the [Graphics repository & Yamato FAQ](http://go/graphics-yamato-faq).

### Checklist for PR maker
- [Y ] Have you added a backport label (if needed)? For example, the `need-backport-*` label. After you backport the PR, the label changes to `backported-*`.
- [Y ] Have you updated the changelog? Each package has a `CHANGELOG.md` file.
- [N/A[] Have you updated or added the documentation for your PR? When you add a new feature, change a property name, or change the behavior of a feature, it's best practice to include related documentation changes in the same PR. If you do add documentation, make sure to add the relevant Graphics Docs team member as a reviewer of the PR. If you are not sure which person to add, see the [Docs team contacts sheet](https://docs.google.com/spreadsheets/d/1rgUWWgwLFEHIQ3Rz-LnK6PAKmbM49DZZ9al4hvnztOo/edit#gid=1058860420).
- [N/A ] Have you added a graphic test for your PR (if needed)? When you add a new feature, or discover a bug that tests don't cover, please add a graphic test.

---
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?
"Master Preview" in versions before 8 changed to "Main Preview" from versions 9 onwards, but the TOC was not updated. Now the name of the link and the link itself are fixed in the TOC.

---
### Testing status
N/A

---
### Comments to reviewers
N/A

### CODE REVIEW

File: com.unity.shadergraph/Documentation~/TableOfContents.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
     * [My first Shader Graph](First-Shader-Graph) 
     * [Shader Graph Window](Shader-Graph-Window)  
         * [Blackboard](Blackboard)  
-        * [Master Preview](Master-Preview) 
+        * [Main Preview](Main-Preview)
```
Comment (sharlenet) [inline com.unity.shadergraph/Documentation~/TableOfContents.md:7]: Good catch! I took a look and saw that the title for `Main-Preview.md` (in the document itself) is Master Preview.

I'll ask the Shader Graph team what's the official name. Might need to do a mass find-and-replace in all the pages!

**Review by sharlenet — COMMENTED** [review]
Alex confirmed that it is Main Preview. Could you also change the title on `Main-Preview.md` and any other instances of "Master Preview" in the docs?

File: com.unity.shadergraph/Documentation~/TableOfContents.md (hunk 1 above)
Comment (alindmanUnity) [reply com.unity.shadergraph/Documentation~/TableOfContents.md:7]: replaced all instances i could find

**Review by alindmanUnity — APPROVED** [review]
cancelled tests as it's purely MD changes, no need to use up test machines

**Comment by sebastienlagarde** [comment]
Have this been backported? and is it present on 2021.1?

**Comment by ValGrimm-U3D** [comment]
I thought I had backported this, but it seems not. I shall do this now.

**Comment by berniegp** [comment]
The current docs (https://docs.unity3d.com/Packages/com.unity.shadergraph@11.0) have the broken links to Master-Preview.md. The 10.4 docs are ok however which seems strange.

**Comment by ValGrimm-U3D** [comment]
I have made a note to fix this. It is indeed odd.

**Pull request merged**